public/docs/ts/latest/tutorial/toh-pt5.jade

```diff
@@ -148,172 +168,220 @@ code-example(language="bash").
+  We have to ***tell it where*** by adding a `<router-outlet>` element to the bottom of the template.
+  `RouterOutlet` is one of the <span if-docs="ts">directives provided by</span> the `!{_RouterModuleVsRouterDirectives}`.
   The router displays each component immediately below the `<router-outlet>` as we navigate through the application.
 
   ### Router Links
+
   We don't really expect users to paste a route URL into the address bar.
   We add an anchor tag to the template which, when clicked, triggers navigation to the `HeroesComponent`.
 
   The revised template looks like this:
-+makeExample('toh-5/ts/app/app.component.2.ts', 'template', 'app/app.component.ts (template v1)')(format=".")
-:marked
-  Notice the `routerLink` binding in the anchor tag.
-  We bind the `RouterLink` directive (another of the `RouterModule` directives) to a string
-  that tells the router where to navigate when the user clicks the link.
 
-  Since our link is not dynamic, we define a *routing instruction* with a **one-time binding** to our route **path**.
-  Looking back at the route configuration, we confirm that `'/heroes'` is the path of the route to the `HeroesComponent`.
-.l-sub-section
++makeExcerpt('app/app.component.1.ts', 'template-v2')
+
+block routerLink
   :marked
-    For more dynamic router links, learn about the *link parameters array* in the [Routing](../guide/router.html#link-parameters-array) chapter.
+    Notice the `routerLink` binding in the anchor tag.
+    We bind the `RouterLink` directive (another of the `RouterModule` directives) to a string
+    that tells the router where to navigate when the user clicks the link.
+
+    Since our link is not dynamic, we define a *routing instruction* with a **one-time binding** to our route **path**.
+    Looking back at the route configuration, we confirm that `'/heroes'` is the path of the route to the `HeroesComponent`.
+  .l-sub-section
+    :marked
+      Learn more about dynamic router links and the *link parameters array*
+      in the [Routing](../guide/router.html#link-parameters-array) chapter.
+
 :marked
-  Refresh the browser.  We see only the app title. We don't see the heroes list.
+  Refresh the browser.  We see only the app title and heroes link. We don't see the heroes list.
+
 .l-sub-section
   :marked
     The browser's address bar shows `/`.
     The route path to `HeroesComponent` is `/heroes`, not `/`.
     We don't have a route that matches the path `/`, so there is nothing to show.
     That's something we'll want to fix.
+
 :marked
-  We click the "Heroes" navigation link, the browser bar updates to `/heroes`,
+  We click the *Heroes* navigation link, the browser bar updates to `/heroes`,
   and now we see the list of heroes. We are navigating at last!
 
   At this stage, our `AppComponent` looks like this.
-+makeExample('toh-5/ts/app/app.component.2.ts',null, 'app/app.component.ts (v2)')
+
++makeExample('app/app.component.1.ts', 'v2', 'app/app.component.ts (v2)')
```

Comment:
Again, I don't see the router.dart import or the `@RouteConfig` definition here.